python/interpret_community/mimic/models/linear_model.py

```diff
@@ -92,7 +164,8 @@ class LinearExplainableModel(BaseExplainableModel):
     :type random_state: int
     """
 
-    def __init__(self, multiclass=False, random_state=DEFAULT_RANDOM_STATE, classification=True, **kwargs):
+    def __init__(self, multiclass=False, random_state=DEFAULT_RANDOM_STATE, classification=True,
```

Comment:
missing docstrings for classification , sparse_data ?

Reply:
nice catch!